utilities/openshift_resources/oc.py

```
@@ -0,0 +1,156 @@
+
+
+def run_oc(
+    *args: str,
+    capture_output: bool = True,
+    timeout: int = 120,
+    check: bool = True,
+) -> subprocess.CompletedProcess[str]:
+    """Run an oc command and return the result."""
+    command = [OC_BINARY, *args]
+    logger.debug(event="run_oc", command=" ".join(command))
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)